### ISSUE TITLE
ETSI TS 103 720 V1.2.1 - distributed vs. centralized MCE architecture

### ISSUE THREAD

**Issue by kuehnhammer** [5G-MAG/Standards#83]
# Context
The depiction of interface M3 between MME and eNodeB is unclear.

# Problem description

<img width="504" alt="image" src="https://github.com/5G-MAG/Standards/assets/1220339/cc791348-22bc-47ec-8faf-7aedd5633194">

TS 136 300 15.1.1 defines interface M3 between MME and MCE, and M2 between MCE and eNodeB. For the logical MCE entity, two deployment options are given: a centralized MCE serving multiple eNodeBs, and a distributed architecture where each eNodeB is deployed with an MCE alongside it (see Figure 15.1.1-2), and mandates that in both cases the M2 interface between MCE and eNodeB be kept.

<img width="499" alt="image" src="https://github.com/5G-MAG/Standards/assets/1220339/065f2bc4-6dcc-4bc2-814d-c1d9a0691614">

The architecture and deployment diagrams in TS 103 720 show a direct M3 interface between eNodeB and MME, which does not match the above.

Additionally, the currently available commercial broadcast core by R&S uses interfaces M1 and M2 for communication with the eNodeB, i.e. it has an integrated MCE. To maintain compatibility with new cores, it would be good if the TS reflects this model as well.

# Suggested solution
Add the MCE component in Figure 5.2.1-1, and possibly adapt Figure 5.2.2-1 to show the MCE as part of the integrated core components with M1 and M2 as RAN interfaces; and/or add another deployment model with distributed MCEs alongside the eNodeB (keeping the M2 interface).

**Comment by rjb1000** [comment]
> The architecture and deployment diagrams in TS 103 720 show a direct M3 interface between eNodeB and MME, which does not match the above.

I disagree slightly with this assessment based on the information provided.

It seems to me that the architecture figures in ETSI TS 103 720 are already fully compliant with the second deployment architecture option in figure 15.1.1-2 of 3GPP TS 136 300. They don't explicitly depict the MCE as a subcomponent of the eNodeB, but it's still there conceptually. In this case, reference point **M2** is subsumed into the composite entity as an internal interface, and is therefore not depicted (just as it doesn't appear in the second deployment architecture of 3GPP TS 136 300).

So, one simple clarification to ETSI TS 103 720 would be to add a figure NOTE to figures 5.2.1-1 and 5.2.2-1 stating:

> NOTE: In this reference architecture, the MCE is assumed to be a subfunction of the eNodeB, as shown in the second deployment architecture in figure 15.1.1-2 of 3GPP TS 136 300 [X]. In this case, reference point M2 is an internal interface that does not need to be exposed by the eNodeB.

Turning to the current implementation:

> the currently available commercial broadcast core by R&S uses interfaces M1 and M2 for communication with the eNodeB, i.e. it has an integrated MCE.

This implies that R&D has chosen to implement something closer to the first deployment model illustrated in figure 15.1.1-2 of 3GPP TS 136 300 in their 5B Broadcast Transmitter product. If we want to document that deployment explicitly in ETSI TS 103 720, I would argue that separate figures are required; modifying the existing ones risks creating more confusion rather than less. There's a reason the two deployment choices are depicted separately in 3GPP TS 136 300, and we should follow the same pattern if we choose to document the alternative deployment architecture.

However, before reaching that decision, it's worth pausing a moment. Looking at this a different way, the R&S implementation has simply chosen to exceed the requirements of ETSI TS 103 720 by needlessly exposing reference point **M2** to the outside world. There's nothing wrong _per se_ with doing more than the specification requires: it's an implementation choice. In terms of the deployment architecture shown in figure 5.2.2-1 of ETSI TS 103 720, understood in the context of the second deployment architecture in figure 15.1.1-2 of 3GPP TS 136 300, one could interpret this as the MCE inside the eNodeB offering an **M2**-compliant interface at the boundary of the _5G Broadcast Transmitter (Radio Access Network)_ functional block, but it is just dangling uselessly with nothing connected to it.

From this alternative perspective, no changes are really needed to TS 103 720, other than perhaps the above figure NOTEs for clarification.

**Comment by rjb1000** [comment]
Based on discussion with @kuehnhammer on today's 5G-MAG Reference Tools call, he has convinced me that the figures in TS 103 720 are incorrect with respect to the reference model for eMBMS.

- The choice of the left hand deployment architecture by R&S seems valid.
  - It potentially allows a single _5G Broadcast Transmitter (Core)_ to manage multiple _5G Broadcast Transmitter (Radio Access Network)_ instances.
- The choice of the right hand deployment architecture seems equally valid.

# Reference architecture

Amend figure 5.2.1-1 to explicitly depict an MCE between the MME and the eNodeB.

**M3** must be shown between the MME and the MCE.

<img width="80%" align="center" alt="TS 103 720 figure 5 2 1-1 2023-08-11" src="https://github.com/5G-MAG/Standards/assets/9865771/3649ddb2-e231-47c2-b03d-beb8fe0404dd">

# Two deployment options

Replace figure 5.2.2-1 with two deployment options.

## **M2** exposed between Core and RAN

This corresponds to the R&S deployment choice. **M3** (between the MME and MCE) is hidden inside the Core.

<img width="80%" align="center" alt="TS 103 720 figure 5 2 2-2 2023-08-11" src="https://github.com/5G-MAG/Standards/assets/9865771/42caf2e3-140b-4701-bf82-259cc3714cec">

## **M3** exposed between Core and RAN

This is a modification of the deployment model already documented in TS 103 720, with **M3** now terminating on an MCE instantiated inside the eNodeB. **M2** is hidden inside the eNodeB.

<img width="80%" align="center" alt="TS 103 720 figure 5 2 2-1 2023-08-11" src="https://github.com/5G-MAG/Standards/assets/9865771/51fa7dd6-43ed-4cae-8303-f8ded9d30865">

**Comment by dvargasparedero** [comment]
comments based on WG CD:
- @haudiobe to follow up on this issue

**Comment by jordijoangimenez** [comment]
- Check with @johannmika if this is worth to be part of a new clause 5.2.2.3

**Comment by rjb1000** [comment]
Uploaded [**CR0083**rev1](https://member.5g-mag.com/wg/CD-T/document/1459?downloadRevision=3031) with the figure for distributed MCE deployment revised.

**Comment by jordijoangimenez** [comment]
This was merged into https://member.5g-mag.com/wg/CD-T/document/1346
